pbf/views.py

```diff
@@ -817,8 +846,7 @@ def change_name(request, player_id):
 def tab(request, game_id, player_id):
     game = get_object_or_404(Game, pk=game_id)
     player = get_object_or_404(GamePlayer, pk=player_id)
-    compute_card_thresholds(player)
-    return render(request, 'tabs.html', {'game': game, 'player': player})
+    return render_player_view(request, player)
```

Comment:
`tabs.html` should not be replaced with the player view. It would make it impossible to choose between multiple players. Add two spirits to a game and see that the tabs are missing.

Reply:
Good catch; last minute change that should've been excluded.